odxtools/parameterinfo.py

```diff
@@ -3,15 +3,21 @@
 from io import StringIO
 from typing import Iterable
 
+from .compumethods.compucodecompumethod import CompuCodeCompuMethod
 from .compumethods.identicalcompumethod import IdenticalCompuMethod
 from .compumethods.limit import IntervalType
 from .compumethods.linearcompumethod import LinearCompuMethod
+from .compumethods.linearsegment import LinearSegment
+from .compumethods.ratfunccompumethod import RatFuncCompuMethod
+from .compumethods.ratfuncsegment import RatFuncSegment
+from .compumethods.scalelinearcompumethod import ScaleLinearCompuMethod
+from .compumethods.scaleratfunccompumethod import ScaleRatFuncCompuMethod
```

Comment:
Too many changes in this file, that are unrelated to PR topic, please move them to their own PR

Reply:
I moved the changes for `free_parameter_info()` into a separate PR: #348